src/main/java/com/algorand/algosdk/algod/client/StringUtil.java

```diff
@@ -13,7 +13,7 @@
 
 package com.algorand.algosdk.algod.client;
 
-
+@Deprecated
 public class StringUtil {
```

Comment:
nit: I have some different thoughts on it. If you look at this class's usage, then it gets into ABI implementation of some string concatenation. Shell we mark this one as deprecated, and *also take care of the ABI case*?

Reply:
Good observation - I think we should use `org.apache.commons.lang3.StringUtils` since we have already added that dependency in `src/main/java/com/algorand/algosdk/v2/client/Utils.java`. 

Otherwise, we *could* use the `join` in `src/main/java/com/algorand/algosdk/kmd/client/StringUtil.java`, since `kmd` is not being deprecated. I think using `commons.lang3.StringUtils` is a better option though, and if you agree, I can make the change in this PR in the ABI classes.

Reply:
Great - If `org.apache.commons.lang3.StringUtils` is available, then we should change the dependencies in ABI classes, and I think we can include these changes inside this PR, no need to open yet another PR.